CMakeLists.txt

```diff
@@ -29,6 +29,7 @@ set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
 option(ONEDPL_FPGA_STATIC_REPORT "Enable the static report generation for the FPGA device" OFF)
 option(ONEDPL_USE_AOT_COMPILATION "Enable the ahead of time compilation" OFF)
 option(ONEDPL_ENABLE_SIMD "Enable SIMD vectorization by passing an OpenMP SIMD flag to the compiler if supported" ON)
+option(ONEDPL_WORKAROUND_FOR_IGPU_64BIT_REDUCTION "Avoid using known identity for 64-bit arithmetic data types")
```

Comment:
By default, the macro `ONEDPL_WORKAROUND_FOR_IGPU_64BIT_REDUCTION` is undefined.
@dmitriy-sobolev, iIs it OK to declare this option without initial value?

Reply:
From my point of view, we are unable to do this because by default we we shouldn't define `ONEDPL_WORKAROUND_FOR_IGPU_64BIT_REDUCTION` macro to ay state.

Reply:
You are free to leave it as is. Note, that  `ONEDPL_WORKAROUND_FOR_IGPU_64BIT_REDUCTION` CMake option will be `OFF` whether its is explicitly set or omitted.

I believe you do not confuse `ONEDPL_WORKAROUND_FOR_IGPU_64BIT_REDUCTION` as a macro and a CMake option: these are different entities.